packages/plans-grid-next/src/components/shared/header-price/index.tsx

```diff
@@ -14,12 +21,27 @@ interface HeaderPriceProps {
 	visibleGridPlans: GridPlan[];
 }
 
+const getTermVariantPlanSlugForSavings = ( {
+	planSlug,
+	billingPeriod,
+}: {
+	planSlug: PlanSlug;
+	billingPeriod?: -1 | ( typeof PERIOD_LIST )[ number ];
+} ) => {
+	if ( billingPeriod && 365 <= billingPeriod ) {
```

Comment:
@jeyip I think we need to decide here what the savings are "from". Is it everything against the monthly price, or yearly off monthly and then 2/3-yearly off the yearly price?

Reply:
Or is it everything off the immediately lower term's? So yearly against monthly, then 2-yearly against yearly, etc.

Reply:
> Is it everything against the monthly price, or yearly off monthly and then 2/3-yearly off the yearly price?

> Or is it everything off the immediately lower term's? So yearly against monthly, then 2-yearly against yearly, etc.

Considering this more. I know we've been talking about the immediate lower term's price, but I actually think it would make more sense to match the billing term selector, which shows percentage discounts against the price of monthly plans carried over their respective term.

For example, the billing term selector for 2 year plans show up to a 63% discount.

<img width="401" alt="Screenshot 2024-11-17 at 16 06 12" src="https://github.com/user-attachments/assets/9f8b9696-ad4c-4ac8-9ad4-9180809d1cf2">

The calculation is done by comparing a 2 year premium plan at $156 vs 24 months of a monthly premium plan $432. By extension, in our case, the $18 monthly price as the would be thecrossed out price. It seems natural to have, like you've said, everything against the monthly price